app/Models/DBC/Area/AreasAndZones.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models\DBC\Area;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * App\Models\DBC\Area\AreasAndZones
+ *
+ * @property integer $m_ID
+ * @property integer $m_ParentAreaID
+ * @property string $m_AreaName_lang
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMID($value)
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMParentAreaID($value)
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMAreaNameLang($value)
+ * @mixin \Eloquent
+ */
+class AreasAndZones extends Model
```

Comment:
And now, getters and setters isn`t laravel style, u can read about it. If u need getters and setters for params u can read about mutators: https://laravel.com/docs/5.2/eloquent-mutators